invenio_rdm_records/services/config.py

```diff
@@ -428,6 +428,7 @@ class RDMRecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
         ),
         "versions": RecordLink("{+api}/records/{id}/versions"),
         "access_links": RecordLink("{+api}/records/{id}/access/links"),
+        "access_grants": RecordLink("{+api}/records/{id}/access/grants"),
         "access_users": RecordLink("{+api}/records/{id}/access/users"),
```

Comment:
just for me to understand, aren't users and grands endpoints the same? or it is planned for grants to return groups as well ?

Reply:
`access_grants` is an endpoint for getting `grant` object itself - altogether with `users` and `roles`. 
`access_users ` and afterwards `access_groups` will be subject specific. These 2 have the same resource, but different resource configs and different endpoints